src/core/smoothing/none.rs

```diff
@@ -4,16 +4,15 @@
 use super::*;
 use crate::gyro_source::TimeQuat;
 
+
 #[derive(Clone)]
 pub struct None {
-    pub horizonlockpercent: f64,
-    pub horizonroll: f64,
+    pub horizonlock: horizon::HorizonLock
 }
 
 impl Default for None {
     fn default() -> Self { Self {
-        horizonlockpercent: 0.0,
-        horizonroll: 0.0,
+        horizonlock: horizon::HorizonLock::default()
     } }
 }
```

Comment:
This can be replaced by simply `#[derive(Clone, Default)]` just before `pub struct None`